### PR TITLE
Added cancer hallmark full publications details (rows)

### PR DESCRIPTION
**Pull request by LucaFumis**
Added plain `rows` to cancerHallmarks query.

### CODE REVIEW

File: src/apis/dataloaders.js [hunk 1]
```diff
@@ -343,6 +343,15 @@ export const createTargetLoader = () =>
             roleInCancer: (hallmarks ? hallmarks.attributes : [])
               .filter(d => d.attribute_name === 'role in cancer')
               .map(d => ({ name: d.description, pmId: d.pmid })),
+            rows: (hallmarks ? hallmarks.cancer_hallmarks : [])
+              .map(d => ({
+                name: d.label,
+                description: d.description,
+                promotes: d.promote,
+                suppresses: d.suppress,
+                activity: d.promote ? 'promotes' : (d.suppress ? 'suppresses' : ''),
```
Comment (peatroot) [inline src/apis/dataloaders.js]: Maybe `activity` is redundant? It seems to be derived from `promotes` and `suppresses`.

File: src/schema/details/CancerHallmarks.js [hunk 2]
```diff
@@ -11,9 +11,18 @@ export const typeDefs = gql`
...
+    promotes: Boolean!
+    suppresses: Boolean!
+    activity: String!
+    description: String!
+    pmId: String!
+  }
   type TargetDetailCancerHallmarks {
     publicationsByHallmark: [CancerHallmarkPublicationsByHallmark!]!
     roleInCancer: [CancerHallmarkRoleInCancer!]!
+    rows: [CancerHallmarkPublicationFullDetails!]
```
Comment (peatroot) [inline src/schema/details/CancerHallmarks.js]: Probably want `rows: [CancerHallmarkPublicationFullDetails!]!`.

**Review by peatroot — CHANGES_REQUESTED** [review]
Looks good. Minor tweaks suggested.

File: src/apis/dataloaders.js (hunk 1 above)
Comment (LucaFumis) [reply src/apis/dataloaders.js]: Addressed in last commit https://github.com/opentargets/platform-api/pull/19/commits/6152f197ae651b794e008781b4d93d22f194a849

File: src/schema/details/CancerHallmarks.js (hunk 2 above)
Comment (LucaFumis) [reply src/schema/details/CancerHallmarks.js]: Addressed in last commit https://github.com/opentargets/platform-api/pull/19/commits/6152f197ae651b794e008781b4d93d22f194a849

**Pull request merged**